### DISCUSSION TITLE
Roland's MSEide installer

### DISCUSSION THREAD

**Discussion by rchastain** [mse-org/mseide-msegui#71] [Show and tell]
Hello friends!

I made an [installer](https://gitlab.com/rchastain/msegui-studies/-/tree/master/mymseinstaller?ref_type=heads) (for now Linux only).

![screenshot](https://github.com/mse-org/mseide-msegui/assets/13957204/8e21c254-c252-4088-be61-41bba13e7fb6)

When you have time, please tell me what you think about it.

I also made a script (for Linux too). See attached file.

The installer and the script do more or less the same job, excepted that the installer clones the git repository, while the script downloads the ZIP file of a given version (5.10.0).

It shouldn't bee too difficult to make the installer work under Windows too.

Regards.

Roland

[installmseide5100.zip](https://github.com/mse-org/mseide-msegui/files/14300511/installmseide5100.zip)

**Comment by fredvs** [comment]
> [installmseide5100.zip](https://github.com/mse-org/mseide-msegui/files/14300511/installmseide5100.zip)

WoW, out-of-the-box and magical.
The script worked perfectly here on Linux X86_64.

And added a beautiful MSEgui desktop icon.

Well done Roland!

**Comment by fredvs** [comment]
The boring explorer is back again...

Suggestion: also add **MSEide** in the "Development" section with a nice icon: (see picture):

![roland_script_install](https://github.com/mse-org/mseide-msegui/assets/3421249/48ceccfd-45a5-4494-86e7-6d8dce7f7c5a)

**Comment by fredvs** [comment]
Ooops, ok, you place it into "Other".

![roland_script_install2](https://github.com/mse-org/mseide-msegui/assets/3421249/d1d7ada8-4282-4b13-94fe-7fdbb9783948)

Sorry for the noise.

**Comment by rchastain** [comment]
You are right Fred: I forgot to fill the **categories** field in _.desktop_ files (in the installer and also in the script). I will add it tomorrow.

Thanks for your feedback.

**Comment by rchastain** [comment]
I think we can add this line:

```
Categories=Development;IDE;GUIDesigner;
```

**Comment by rchastain** [comment]
Here is the corrected version of the script (with **Categories** and **Keywords** keys added in _.desktop_ file).

[installmseide5100-categories.zip](https://github.com/mse-org/mseide-msegui/files/14396109/installmseide5100-v2.zip)

There is also a new version of the [GUI installer](https://gitlab.com/rchastain/msegui-studies/-/tree/master/mymseinstaller?ref_type=heads).

**Comment by fredvs** [comment]
Hello Roland!

Perfect:

![roland_installer](https://github.com/mse-org/mseide-msegui/assets/3421249/4a444101-83fd-4544-8bb5-52e7cf65953c)

Tip: Maybe, before to zip the script, set it as executable so it is more out-of-the-box:

`$ chmod u+x installmseide5100.sh`

I vote to use the **Roland,s MSEide installer** for releases of MSEide.

Well done Roland!

Fre;D

**Reply by rchastain** [reply]
Glad that it works for you. Thanks for the tip about making the script executable.
